docs/design/windows-design.md

```diff
@@ -210,8 +210,17 @@ Kube-proxy userspace mode is configured to provide NodePort Service function. A
 "HNS Internal NIC" is provided to kube-proxy to configure Service addresses. The OpenFlow entries for the
 NodePort Service traffic on Windows are the same as those on Linux.
 
-The ClusterIP Service function is implemented by Antrea leveraging OVS. Antrea installs OpenFlow entries
-to select the backend endpoint and performs DNAT on the traffic.
+The ClusterIP Service function is implemented by AntreaProxy. Antrea installs routes from ClusterIP Services
+to a virtual IP (169.254.169.252) and a route from the virtual IP to antrea-gw0. There's an ARP responder flow
```

Comment:
Move ARP responder flow to end of the paragraph. Check my suggested version below.

Reply:
Updated.